discovery-provider/src/api/v1/transactions.py

```diff
@@ -76,10 +76,10 @@ class GetTransactionHistory(Resource):
     @full_user_ns.marshal_with(transaction_history_response)
     @auth_middleware()
     def get(self, id, authed_user_id=None):
-        self._get(id, authed_user_id)
-
-    def _get(self, id, authed_user_id=None):
         user_id = decode_with_abort(id, full_user_ns)
```

Comment:
would we still be decoding an already decoded id here?

Reply:
the other usage is the one that has it already decoded:
https://github.com/AudiusProject/audius-protocol/blob/211cf0fff6a72953d85dd65ef55cc83c43216963/discovery-provider/src/api/v1/transactions.py#L114